### PR TITLE
Few more itmes in carrion uplink

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Adds the surgery kit, shipwide announcement and the chameleon kit to the carrrion uplink. 

## Why It's Good For The Game

Carrions get to use a few more fun and stealthy items if they don't want any more gene points, the surgery kit should also make organ extraction less painful.

## Changelog
:cl:
add: Added the surgery kit, the chameleon kit and the shipwide announcement to the carrion uplink.
/:cl:

### CODE REVIEW

File: code/datums/uplink/announcements.dm [hunk 1]
```diff
@@ -14,6 +14,7 @@
 	name = "Shipwide Announcement"
 	item_cost = 2
 	desc = "Broadcasts a message anonymously to the entire vessel. Triggers immediately after supplying additional data."
+	antag_roles |= ROLE_CARRION
```
Comment (MadmanMartian) [inline code/datums/uplink/announcements.dm:17]: I really don't think this is how you do it

**Review by MadmanMartian — CHANGES_REQUESTED** [review]
The "fake" shipwide announcements button is currently very broken, being able to trigger random events at will.

If you're going to add that to another antag, fix it so it's not an instant radstorm button.

**Comment by TheShown911** [comment]
Isn't that the fake radiation announcement from lines 92/93?

File: code/datums/uplink/announcements.dm (hunk 1 above)
Comment (TheShown911) [reply code/datums/uplink/announcements.dm]: Fixed.

**Comment by TheShown911** [comment]
Update: Should also fix the bug of serb ship being unable to move after an antagonist uses the fake serb announcement.

**Review by MadmanMartian — APPROVED** [review]

**Pull request merged**